### PR TITLE
Add new RdsStartExportTaskOperator parameters

### PR DESCRIPTION
**Pull request by covidium**
Add support to specify max_attempts and check_interval for RdsStartExportTaskOperator

---
**^ Add meaningful description above**
Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

File: airflow/providers/amazon/aws/operators/rds.py [hunk 1]
```diff
@@ -351,6 +351,8 @@ def __init__(
         s3_prefix: str = "",
         export_only: list[str] | None = None,
         wait_for_completion: bool = True,
+        waiter_interval: int = 30,
+        waiter_max_attempts: int = 40,
```
Comment (vincbeck) [inline airflow/providers/amazon/aws/operators/rds.py:357]: Please add these to docstring as well

Comment (covidium) [reply airflow/providers/amazon/aws/operators/rds.py:357]: Added

**Review by vincbeck — APPROVED** [review]

File: airflow/providers/amazon/aws/operators/rds.py [hunk 2]
```diff
@@ -351,6 +353,8 @@ def __init__(
...
```
Comment (o-nikolas) [inline airflow/providers/amazon/aws/operators/rds.py:357]: 20 minutes by default feels like a long time. Is that reasonable for this context?

Comment (covidium) [reply airflow/providers/amazon/aws/operators/rds.py:357]: That's the default waiting time defined in the hook wait_for_export_task_state. I've added these settings as in us-east-1 we're getting a 20 minutes delay before the actual export task starts, resulting in the failure of the DAG, so I need a way to modify those values and increase them over 30 minutes.

Comment (vincbeck) [reply airflow/providers/amazon/aws/operators/rds.py:357]: Agree, we have to set these values to be backward compatible

Comment (o-nikolas) [reply airflow/providers/amazon/aws/operators/rds.py:357]: Cool, fair enough, lgtm!

**Review by o-nikolas — APPROVED** [review]

**Pull request merged**